.gitignore

```diff
@@ -11,3 +11,4 @@ tests/testdata/app_data/
 docs
```

Comment:
Hi @randrescastaneda , do you know why we have `docs` in `.gitignore` ? Is it intentional or is done by mistake? If we push `docs` folder to Github then we can publish the vignette in package documentation.

Reply:
Hi @shahronak47 , 

No, no idea. @tonyfujs , Do you know why we have `docs` in `.gitignore`? Does it affect the api in the server?

Thanks.

Reply:
Mmm... That does not ring any bell. No idea...

Reply:
thanks for quick response both. I am going to remove this and in case, something breaks then we revert it back.